### ISSUE TITLE
dec_lap_pyr parameter ?

### ISSUE THREAD

**Issue by lulu1315** [nkolkin13/STROTSS#3]
hello nick.
in pyr_lap.py the function dec_lap_pyr takes levs as a parameter.
in st_helper you call the function with levs=5
when using more levels than 5 (as in your defaults) should levs be set to max_scl ?
thanks in advance
luc

**Comment by nkolkin13** [comment]
Hi Luc,
No, back-propagating through the pyramid coefficients is just a trick to allow the image to be altered more quickly by gradient descent. The number of levels the pyramid has is just a hyperparameter that doesn't need to be tied to anything else in the algorithm.
Best,
Nick

**Comment by lulu1315** [comment]
ok.thank you Nick.
